config/rewrites.d/docs.rackspace.com.json

```diff
@@ -10,9 +10,18 @@
 			"status": 302
 		},
 		{
-			"description": "cloud servers legacy",
-			"from": "^/servers/api/v1.0/cs-devguide",
-			"to": "/docs/#docs-cloud-servers-legacy",
+			"description": "autoscale",
+			"from": "^/cas/api/v1.0/autoscale-devguide/?(.*)$",
+			"to": "/docs/autoscale/v1/developer-guide/#document-index",
```

Comment:
Actually, one other comment: I think we can remove the v10 redirect, as we no longer publish v10 docs.